File: ignored-test/CSharpXUnit/CSharpXUnit.Tests/Tests/Create_ReturnsToken.cs
using Xunit;

namespace DefaultNamespace
{
    public class TokenControllerTests
    {
        [Fact(Skip = "")]
        public void Create_ReturnsToken()
        {
            {
                var vm = new TokenViewModel
                {
                    Email = "admin@contoso.com",
                    Password = "Pass@word1!"
                };
                var result = await _sut.Create(vm);
                Assert.IsType<OkObjectResult>(result);
                string token = ((OkObjectResult)result).Value.ToString();
                Assert.Contains("token", token);
            }
        }
    }
}

File: ignored-test/java-junit/Create_ReturnsToken.java
import org.junit.jupiter.api.Test;
import static org.junit.jupiter.api.Assertions.*;
import static org.mockito.Mockito.*;

class TokenControllerTests {

    @Disabled
    @Test
    void createReturnsToken() throws Exception {
        TokenViewModel vm = new TokenViewModel();
        vm.setEmail("admin@contoso.com");
        vm.setPassword("Pass@word1!");
        Object result = _sut.create(vm);
        assertTrue(result instanceof OkObjectResult);
        String token = ((OkObjectResult) result).getValue().toString();
        assertTrue(token.contains("token"));
    }
}
